server/routes/user.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const User = require('../database/models/user');
+const registrationValidator = require('../controllers/userRegistrationValidator');
+const loginValidator = require('../controllers/userLoginValidator');
+
+router.post('/register', (req, res) => {
+  const {validationErrors, isRegistrationValid} = registrationValidator(req.body);
+
+  if(!isRegistrationValid) {
+    return res.status(400).json(validationErrors);
+  }
+
+  User.findOne({ email: req.body.email })
+      .then(user => {
+        if(user) {
+          return res.status(400).json({email: "Email you entered already exists"});
+        }
+        else {
+          const newUser = new User({
```

Comment:
should registration also include the language? or is it not there bc requirements have not been defined?

Reply:
Yes it should and in model it does. I missed it completely here so thanks for catching that. It was working because by default language English was being applied automatically. Will be fixed!